integration-tests/common-test-code/src/main/java/com/vaadin/flow/quarkus/it/routecontext/MasterView.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2000-2018 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.vaadin.flow.quarkus.it.routecontext;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.router.AfterNavigationEvent;
+import com.vaadin.flow.router.AfterNavigationObserver;
+import com.vaadin.flow.router.Route;
+import com.vaadin.flow.router.RoutePrefix;
+import com.vaadin.flow.router.RouterLayout;
+import com.vaadin.flow.router.RouterLink;
+import com.vaadin.quarkus.annotation.RouteScopeOwner;
+
+@Route("")
+@RoutePrefix("master")
+public class MasterView extends AbstractCountedView
+        implements RouterLayout, AfterNavigationObserver {
+
+    public static final String ASSIGNED = "assigned";
+    public static final String ASSIGNED_BEAN_LABEL = "ASSIGNED";
+    public static final String APART = "apart";
+    public static final String APART_BEAN_LABEL = "APART";
+
+    @Inject
+    @RouteScopeOwner(MasterView.class)
+    private AssignedBean assignedBean;
```

Comment:
For this 'private' modifier my IDE gives a warning:
Private access modifier usage is not recommended for Quarkus CDI members.
```suggestion
   AssignedBean assignedBean;
```

Reply:
done